packages/photon/README.md

```diff
@@ -2,19 +2,37 @@
 
 JavaScript library for the [WordPress.com][] [Photon][] image manipulation service.
 
+## Requirements
+
+Photon.js requires support for the standard URL and URLSearchParams APIs.
+Be sure to use a polyfill if you're targetting environments without support for them. This includes old browsers, such
+as Internet Explorer 11, and old versions of Node.js (5 or older).
```

Comment:
Would be good to mention this [in the changelog](https://github.com/Automattic/wp-calypso/blob/d265dcd5e08be2d0e26fdfb888267691fb483492/packages/photon/History.md)

Reply:
Oh, I didn't see that file when looking for a changelog. Thanks!

Reply:
Done.